src/app/core/component/graph-display/graph-display.component.ts

```diff
@@ -170,9 +172,30 @@ export class GraphDisplayComponent implements AfterViewInit, OnChanges {
         });
     }
 
+    dropPattern(event: CdkDragDrop<any[]>) {
```

Comment:
to be consistent with our other events, I'd like to use past tense (patternDropped/droppedPattern)